handler/src/main/java/io/netty/handler/logging/LoggingHandler.java

```diff
@@ -226,6 +226,14 @@ public void deregister(ChannelHandlerContext ctx, ChannelPromise promise) throws
     }
 
     @Override
+    public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
+        if (logger.isEnabled(internalLevel)) {
+            logger.log(internalLevel, format(ctx, "RECEIVE COMPLETE"));
```

Comment:
yeah i copied that one. to be honest, i'd prefer they were both READ, but wasn't sure about messing with the behavior.